03 Writing Algorithms/03 Securities/06 Filtering Data/02 Set Models.html

```diff
@@ -0,0 +1,19 @@
+<p>To set a data filter for a security, call the <code>SetDataFilter</code> property on the <code>Security</code> object.</p>
+
+<div class="section-example-container">
+    <pre class="csharp">// In Initialize
+var spy = AddEquity("SPY");
+spy.SetDataFilter(new SecurityDataFilter());</pre>
+    <pre class="python"># In Initialize
+spy = AddEquity("SPY")
+spy.SetDataFilter(SecurityDataFilter())</pre>
+</div>
+
+<p>You can also set the data filter model in a <a href='/docs/v2/writing-algorithms/initialization#07-Set-Security-Initializer'>security initializer</a>. If your algorithm has a universe, use the security initializer technique. In order to initialize single security subscriptions with the security initializer, call <code>SetSecurityInitializer</code> before you create the subscriptions.</p>
+
+<div class="section-example-container">
```

Comment:
Replace this code snippet with an import like https://github.com/QuantConnect/Documentation/blob/master/03%20Writing%20Algorithms/24%20Reality%20Modeling/06%20Buying%20Power/05%20Set%20Models.php#L14-L18 so that we don't overwrite the default reality models of the brokerage model.